common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationSettings.java

```diff
@@ -409,4 +411,20 @@ public void setDisableWebViewHardwareAcceleration(boolean enable) {
     public boolean getDisableWebViewHardwareAcceleration() {
         return mEnableHardwareAcceleration;
     }
+
+    /**
+     * Method to suppress errors where predefinedKeyLoader is not found to decrypt the cache content
+     * @param shouldIgnorePredefinedKeyLoaderNotFoundError if true, suppresses the error
+     */
+    public void setShouldIgnorePredefinedKeyLoaderNotFoundError(boolean shouldIgnorePredefinedKeyLoaderNotFoundError) {
```

Comment:
just a nit: Assuming this is a new method, looking at the method name, it's long. 
If the intent is to ignore cases where predefined key is not set, would "IgnorePredefinedKeyNotFoundErrors"

Reply:
Updated to setIgnoreKeyLoaderNotFoundError